onnx/test/parser_test.py

```diff
@@ -218,6 +218,38 @@ def test_parse_node(self):
         self.assertEqual(node.domain, "SomeDomain")
         self.assertEqual(node.op_type, "SomeOp")
 
+    @parameterized.expand(
+        [
+            (
+                "not_a_good_float"
```

Comment:
```suggestion
                "not_a_good_float",
```

Add a trailing comma here and below to make them tuples